appdatabase/migrations/sql/1685109694_add_customisation_color_id.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE keypairs_accounts ADD COLUMN colorId VARCHAR NOT NULL DEFAULT "";
+UPDATE keypairs_accounts SET colorId = "";
```

Comment:
@Khushboo-dev-cpp map old colors to new color ids in this migration script, just to be a backward compatible.